src/ui/ui.css

```diff
@@ -31,11 +31,11 @@ body {
   animation: mspin-rotate 1568.63ms infinite linear
 }
 
-.swg-loading > div {
+swg-loading > div {
```

Comment:
Let's rename this `div` as well. E.g. `swg-loading-dot` or `-item`. On the other hand - totally fine contracting them in any possible way.

Reply:
Done